src/ol/imagetile.js

```diff
@@ -97,7 +97,7 @@ ol.ImageTile.prototype.getKey = function() {
 ol.ImageTile.prototype.handleImageError_ = function() {
   this.state = ol.TileState.ERROR;
   this.unlistenImage_();
-  this.image_ = ol.ImageTile.blankImage;
+  this.image_.src = ol.ImageTile.blankImage.toDataURL('image/png');
```

Comment:
It would be more efficient to call `toDataURL()` only once. This means instead of creating an `ol.ImageTile.blankImage`, we would create an `ol.ImageTile.blankImageUrl`.